src/components/Archive/AddArchive/PickWeather.tsx

```diff
@@ -0,0 +1,89 @@
+import { useState } from "react";
+
+type Weather = "sun" | "cloud" | "rain" | "snow" | null;
+
+const PickWeather = () => {
+  const [selectedEmoji, setSelectedEmoji] = useState<Weather>(null);
+  const handleWeather = (emoji: Weather) => {
+    setSelectedEmoji(emoji);
+  };
+  const [isDropdownView, setDropdownView] = useState(false);
+  const handleClickContainer = () => {
+    setDropdownView(!isDropdownView);
```

Comment:
이전 상태를 이용해서 현재 상태를 변경하는 경우에는 콜백 함수를 이용해서 상태를 변경해주시면 좋을 것 같아요!
```suggestion
    setDropdownView(prev => !prev);
```